discretisedfield/mesh.py

```diff
@@ -380,11 +430,11 @@ def __iter__(self):
             yield self.index2point(index)
 
     def axis_points(self, axis, /):
-        warnings.warn("Deprecated; use `mesh.midpoints` instead.", FutureWarning)
-        return getattr(self.midpoints, axis)
+        warnings.warn("Deprecated; use `mesh.points` instead.", FutureWarning)
+        return getattr(self.points, axis)
```

Comment:
I would remove this now (maybe raise an AttributeError which points to `points`).